share/tools/web_config/webconfig.py

```diff
@@ -82,6 +82,11 @@ def is_wsl():
     return False
 
 
+def is_win():
```

Comment:
I would prefer if this were called "is_windows". We don't need to save the characters and that makes it easier to read and see.

Reply:
done